test/data/extB/views/node_modules/@oat-sa/some-fake-library/dist/another-fake-file.js

```diff
@@ -0,0 +1,7 @@
+define([
+    'css!extB/some-fake-lib/css/some-fake-style',
```

Comment:
Maybe you can extend the test with a dependency that contains `.css` and `.tpl` in the end to be sure it will work as well.

Reply:
For now, this covers the main functionality. Can be added later.